src/LinearOperators.jl

```diff
@@ -53,6 +54,32 @@ mutable struct LinearOperator{T} <: AbstractLinearOperator{T}
   prod    # apply the operator to a vector
   tprod   # apply the transpose operator to a vector
   ctprod  # apply the transpose conjugate operator to a vector
+  nprod :: Int
+  ntprod :: Int
+  nctprod :: Int
+end
+
+LinearOperator{T}(nrow::Int, ncol::Int, symmetric::Bool, hermitian::Bool, prod, tprod, ctprod) where T =
+  LinearOperator{T}(nrow, ncol, symmetric, hermitian, prod, tprod, ctprod, 0, 0, 0)
+
+nprod(op::AbstractLinearOperator) = op.nprod
+ntprod(op::AbstractLinearOperator) = op.ntprod
+nctprod(op::AbstractLinearOperator) = op.nctprod
+
+increase_nprod(op::AbstractLinearOperator) = (op.nprod += 1)
+increase_ntprod(op::AbstractLinearOperator) = (op.ntprod += 1)
+increase_nctprod(op::AbstractLinearOperator) = (op.nctprod += 1)
```

Comment:
I think you need to define it for Adjoint, Transpose and Conjugate as well

Reply:
You're right. It's a non-issue at this point because we're not wrapping operators into other operators, but it will be done in the next PR.